src/components/Content/content.jsx

```diff
@@ -3,42 +3,55 @@ import './content.css'
 
 import Grid from './Grid/grid';
 import CarouselField from './Carousel/carousel';
+import ProductList from './ProductList/productList';
 
 import featuredProducts from '../../mocks/es-mx/featured-products.json';
 import banners from '../../mocks/es-mx/featured-banners.json';
 import productsCategories from '../../mocks/es-mx/product-categories.json';
 
+
+
 const Content = (props) => {
     const products = featuredProducts.results;
     const carouselProducts = banners.results;
     const carouselCategories= productsCategories.results;
-    console.log(props);
     
     return ( 
         <div className='content-container'>
-            <div className='welcome-container'>
-                <div className='vertical-carousel'>
-                    <CarouselField
-                        products={carouselProducts}
-                    />
+            {props.allProducts? 
+                <ProductList
+                    categories ={carouselCategories}
+                />
+            : 
+            <div className='home-page'>                       
```

Comment:
Haciendo los cambios que comentaba este componente podría verse así
```suggestion
            {allProducts ? 
                <ProductList
                    categories ={carouselCategories}
                />
            :  <HomeContent />
            }
```
Faltaría pasarle los props adecuados pero algo así seria la estructura